liquid-base/liquid-base.cabal

```diff
@@ -257,4 +257,4 @@ library
                       NoImplicitPrelude
   if impl(ghc >= 8.10)
     ghc-options: -fplugin=LiquidHaskell -fplugin-opt=LiquidHaskell:--no-positivity-check
-  ghc-options:        -j
```

Comment:
Is this an unrelated change?

Reply:
No, making changes to the pragma code revealed that such flags will crash liquid even when they didn't before.  So I'm removing them everywhere.